openfe/setup/molecule.py

```diff
@@ -1,10 +1,12 @@
 # This code is part of OpenFE and is licensed under the MIT license.
 # For details, see https://github.com/OpenFreeEnergy/openfe
-
+import warnings
+with warnings.catch_warnings():
```

Comment:
add a comment as to what the warning you're catching is? (so we know when we can remove this)